app/scripts/lib/rpc-method-middleware/handlers/request-accounts.test.ts

```diff
@@ -155,127 +127,25 @@ describe('requestEthereumAccountsHandler', () => {
     });
 
-    it('sets the approved chainIds on an empty CAIP-25 caveat with isMultichainOrigin: false if origin is not snapId', async () => {
-      const { handler } = createMockedHandler();
-
-      await handler(baseRequest);
-      expect(MockMultichain.setPermittedEthChainIds).toHaveBeenCalledWith(
-        {
-          requiredScopes: {},
-          optionalScopes: {},
-          isMultichainOrigin: false,
-        },
-        ['0x1', '0x5'],
-      );
-    });
-
-    it('sets the approved accounts on the CAIP-25 caveat after the approved chainIds if origin is not snapId', async () => {
-      const { handler } = createMockedHandler();
-
-      MockMultichain.setPermittedEthChainIds.mockReturnValue({
-        requiredScopes: {},
-        optionalScopes: {},
-        sessionProperties: { caveatValueWithEthChainIdsSet: true },
-        isMultichainOrigin: false,
-      });
-
-      await handler(baseRequest);
-      expect(MockMultichain.setEthAccounts).toHaveBeenCalledWith(
-        {
-          requiredScopes: {},
-          optionalScopes: {},
-          sessionProperties: { caveatValueWithEthChainIdsSet: true },
-          isMultichainOrigin: false,
-        },
-        ['0xdeadbeef'],
-      );
-    });
-
-    it('does not set the approved chainIds on an empty CAIP-25 caveat if origin is snapId', async () => {
-      const { handler } = createMockedHandler();
-
-      await handler({ ...baseRequest, origin: 'npm:snap' });
-      expect(MockMultichain.setPermittedEthChainIds).not.toHaveBeenCalled();
-    });
-
-    it('sets the approved accounts for the `wallet:eip155` scope with isMultichainOrigin: false if origin is snapId', async () => {
-      const { handler } = createMockedHandler();
-
-      await handler({ ...baseRequest, origin: 'npm:snap' });
-      expect(MockMultichain.setEthAccounts).toHaveBeenCalledWith(
-        {
-          requiredScopes: {},
-          optionalScopes: {
-            'wallet:eip155': {
-              accounts: [],
-            },
-          },
-          isMultichainOrigin: false,
-        },
-        ['0xdeadbeef'],
-      );
-    });
-
-    it('grants a CAIP-25 permission', async () => {
-      const { handler, grantPermissions } = createMockedHandler();
-
-      MockMultichain.setEthAccounts.mockReturnValue({
-        requiredScopes: {},
-        optionalScopes: {},
-        sessionProperties: { caveatValueWithEthAccountsSet: true },
-        isMultichainOrigin: false,
-      });
-
-      await handler(baseRequest);
-      expect(grantPermissions).toHaveBeenCalledWith({
-        [Caip25EndowmentPermissionName]: {
-          caveats: [
-            {
-              type: Caip25CaveatType,
-              value: {
-                requiredScopes: {},
-                optionalScopes: {},
-                sessionProperties: { caveatValueWithEthAccountsSet: true },
-                isMultichainOrigin: false,
-              },
-            },
-          ],
-        },
-      });
-    });
```

Comment:
Forgive me if its obvious but why aren't these tests relevant anymore? I see they would need to be adjusted but aren't they still test cases?

Reply:
they've been moved/adjusted into MetaMaskController